.github/workflows/main.yml

```diff
@@ -11,8 +11,8 @@ jobs:
   build:
     strategy:
       matrix:
-        python: ["3.6", "3.7", "3.8", "3.9"]
-    runs-on: ubuntu-18.04
+        python: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```

Comment:
Note: Python 3.6 eol was 11 months ago. We could start removing it from the matrix.
This will require a full review of the repositories across the org.

Reply:
Would you consider removing it here so that all downstream repositories who consume the next iteration of these pre-built wheels start dropping 3.6 from the matrix?

Reply:
Pushed [5c5ef1e](https://github.com/ome/zeroc-ice-py-github-ci/pull/2/commits/5c5ef1e9a9ad380225176d024eb31912c0f9ad21) to remove 3.6 from the next-generation of pre-built wheels